--- origsrc/csboard-0.9/Makefile.am	2007-09-03 23:07:27.000000000 -0500
+++ src/csboard-0.9/Makefile.am	2008-04-22 22:58:04.671875000 -0500
@@ -197,7 +197,7 @@
 PACKAGE_DEPENDS =							\
 	-pkg:gtk-sharp-2.0						\
 	-pkg:glade-sharp-2.0						\
-	-pkg:rsvg-sharp-2.0
+	-pkg:rsvg2-sharp-2.0
 
 if !WINDOWS_BUILD
 PACKAGE_DEPENDS += -pkg:gconf-sharp-2.0
--- origsrc/csboard-0.9/configure.in	2007-09-03 22:58:01.000000000 -0500
+++ src/csboard-0.9/configure.in	2008-04-22 22:53:29.578125000 -0500
@@ -56,7 +56,7 @@
 PKG_CHECK_MODULES(BASE_DEPENDENCIES,
 gtk-sharp-2.0 >= $GTKSHARP_REQUIRED_VERSION
 glade-sharp-2.0 >= $GTKSHARP_REQUIRED_VERSION
-rsvg-sharp-2.0 >= $GTKSHARP_REQUIRED_VERSION
+rsvg2-sharp-2.0 >= $GTKSHARP_REQUIRED_VERSION
 gconf-sharp-2.0 >= $GTKSHARP_REQUIRED_VERSION
 )
 
